lib/actions/help.js

```diff
@@ -123,6 +123,9 @@ const general = function(_params, options, done) {
 				console.log('\nGlobal Options:\n');
 				print(parse(capitano.state.globalOptions).sort());
 			}
+			console.log(indent('--debug\n'));
```

Comment:
That's cheating! 🚓  But I will join your scheme. 😁

Reply:
Haha yeah. I'll consolidate this when we move the help command to oclif.